wordpress-mobile/WordPress-Android
# [Gutenberg] Hide audio block on free sites

gb-mobile PR: https://github.com/wordpress-mobile/gutenberg-mobile/pull/3129

Hide audio block on free sites.

### Testing

1. Switch to a free/simple site
1. Open editor
1. Tap block inserter
1. Audio Block should not be in the list  
 
#
  
1. Switch to a paid/self-hosted site
1. Open editor
1. Tap block inserter
1. Audio Block should be in the list


PR submission checklist:
